test/irb/test_evaluation.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require "tempfile"
+
+require_relative "helper"
+
+module TestIRB
+  class EchoingTest < IntegrationTestCase
+    def test_irb_echos_by_default
+      write_ruby <<~'RUBY'
+        binding.irb
```

Comment:
For some reason, opening IRB with `IRB.start` never works in integration tests.

```rb
# this hangs and prints nothing
write_ruby <<~'RUBY'
  require "irb"
  IRB.start(__FILE__)
RUBY
```

Similarly, introducing a helper like `run_irb` to make it run `exe/irb` doesn't work either.

So I decided to go with `binding.irb` for now and investigate this further later.